ember-a11y/2016-07/july-06.md

```diff
@@ -0,0 +1,51 @@
+# ember-a11y Meeting Notes - 2016-07-06
+
+## Attendees
+
+- Nathan Hammond
+- Suz Hinton
+- Ben Holmes
+- David Peter
+- Brian Sipple
+- Taylor Solomon
+- Melanie Sumner
+
+## Actions
+
+- [ ] All - Add content to Newsletter Content canvas doc
+- [ ] Nathan - Finish blog post.
+- [ ] Nathan - Transcribe podcast.
+
+## Discussion Notes
+
+### Newsletter
+- Melanie has a design in a CodePen.
+- Created Newsletter Content canvas doc for tracking content to be added throughout the month.
+
+### AT Vendor Partnership.
+- New programming languages and tools, y'all!
+- All in! (Opt out.)
+- Ben, Suz, and Brian directly voiced interest.
+
+### Modern Web Podcast
+
+[Podcast was published.](http://modernweb.podbean.com/e/s02e07-accessibility-for-the-web-meet-these-dedicated-emberjs-community-members/)
+  - Are we still on message?
+  - Has the message changed?
+  - Should it change?
+  - Maybe come up with a clear set of talking points.
+  - Develop consistent language and terminology.
+
+### Ben's British Exit
+
+- [Pairing all month!](http://pair.ember-a11y.com/)
+- Find loads of things to rope him into.
+- [Udacity web accessibility course](https://www.udacity.com/course/web-accessibility--ud891): good.
+
+### Community Engagement
```

Comment:
I think this is when my audio cut out, but I mentioned that I've been exploring ways to make sure the Ember ecosystem has tooling for authoring/integrating accessible SVGs.

I recently took ownership (on NPM right now, anyway) of [broccoli-svgstore](https://github.com/jmarquis/broccoli-svgstore) to get started on this. Also, after a recent pairing session with Ben, we were able to [encourage](https://github.com/svgstore/svgstore/issues/1) getting [this](https://github.com/svgstore) spun up.